cmd/up/group/cmd.go

```diff
@@ -52,10 +50,15 @@ func (c *Cmd) AfterApply(kongCtx *kong.Context) error {
 	}
 	kongCtx.Bind(upCtx)
 
-	if !upCtx.Profile.IsSpace() {
-		// TODO: add legacy support
-		return errors.New("Only Spaces contexts supported for now.")
+	client, err := upCtx.BuildCurrentContextClient()
+	if err != nil {
+		return errors.Wrap(err, "unable to get kube client")
 	}
+
+	// todo(redbackthomson): Assert we are at the space level
+
+	kongCtx.Bind(client)
```

Comment:
kong is unable to bind interfaces via `Bind()` , instead use `BindTo()`
https://github.com/alecthomas/kong/issues/48#issuecomment-583695243

```suggestion
	kongCtx.BindTo(client, (*ctrlclient.Client)(nil))
```